src/cic/time.clj

```diff
@@ -89,6 +92,17 @@
       (t/plus (t/months 1))
       (t/minus (t/days 1))))
 
+(defn month-as-string
+  [date]
+  (f/unparse month-string date))
+
+(defn quarter-following
+  [date]
+  (let [test (t/minus date (t/days 1))
+        quarter (case (t/month test) 1 4 2 4 3 4 4 7 5 7 6 7 7 10 8 10 9 10 10 1 11 1 12 1)
```

Comment:
this case could just be a map look up, and again, I'm a bit confused by the magic numbers